bin/interpreter.sh

```diff
@@ -42,6 +42,9 @@ while getopts "hp:d:l:v" o; do
             . "${bin}/common.sh"
             getZeppelinVersion
             ;;
+        u)
+            ZEPPELIN_SSH_COMMAND="ssh ${OPTARG}@localhost "
+            ;;
         esac
```

Comment:
This requires the login user must exist in the os account and  be able to ssh to localhost.  I am not sure whether this is a good way, but just feel the approach is a little strange compared to the impersonation implementation in hadoop.